ui/app/pages/swaps/searchable-item-list/searchable-item-list.test.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import { renderWithProvider } from '../../../../../test/jest';
+import SearchableItemList from '.';
+
+const createProps = (customProps = {}) => {
+  return {
+    defaultToAll: true,
+    listTitle: 'listTitle',
+    itemsToSearch: [
+      {
+        iconUrl: 'iconUrl',
+        selected: true,
+        primaryLabel: 'primaryLabel',
+        secondaryLabel: 'secondaryLabel',
+        rightPrimaryLabel: 'rightPrimaryLabel',
+        rightSecondaryLabel: 'rightSecondaryLabel',
+      },
+    ],
```

Comment:
The `fuseSearchKeys` prop is require by ListItemSearch, https://github.com/MetaMask/metamask-extension/blob/1e44c34e1e0f7d70ef5841d04eb8da6e9e23b94f/ui/app/pages/swaps/searchable-item-list/list-item-search/list-item-search.component.js#L83

https://github.com/MetaMask/metamask-extension/blob/76a2a9bb8b6ea04025328d36404ac3b59121dfc8/ui/app/pages/swaps/searchable-item-list/searchable-item-list.js#L65-L70

An example is 
```
  "fuseSearchKeys": [
    {
      "name": "name",
      "weight": 0.499
    },
    {
      "name": "symbol",
      "weight": 0.499
    },
    {
      "name": "address",
      "weight": 0.002
    }
  ],
```

Reply:
Thanks! I didn't see `.isRequired` for that specific prop inside `searchable-item-list.js`, but will add the array you mentioned into default props. With future PRs I will be adding testing of different sets of props, not just happy paths.